src/IrisConvert/IrisConvert.php

```diff
@@ -41,11 +42,19 @@ public function checkFileExists($attachment_id)
     {
         $file = get_attached_file($attachment_id);
```

Comment:
Perhaps worth moving this below the `$as3cf_item` check as if that returns true then there's no need to create this variable?